Remove redundant publishing tools

vbuberen opened the pull request:
## :page_facing_up: Context
It seems that `build.gradle` for the project contains a few outdated plugins and configs for publishing, which aren't used anymore. Except it being redundant components, it also confuses other contributors, since recently someone tried to edit script for publishing on Bintray.

## :pencil: Changes
- Removed plugin and script for Bintray publishing.
- Removed plugin for publishing on Maven.

## :warning: Breaking
No issues expected.

cortinico commented on gradle/gradle-mvn-push.gradle:
```diff
@@ -1,66 +0,0 @@
-            githubRepo = POM_GITHUB_REPO
-            githubReleaseNotesFile = "CHANGELOG.md"
-        }
-    }
```
This block can probably go.

cortinico commented on gradle/gradle-mvn-push.gradle:
```diff
@@ -1,66 +0,0 @@
-
-afterEvaluate { proj ->
-
-    install {
```
This block should stay as we need to specify all the parameters for the POM file

vbuberen replied:
Ok, got it. In this case I will update fields of the POM file, since some of them are outdated.

cortinico commented on gradle/gradle-mvn-push.gradle:
```diff
@@ -1,66 +0,0 @@
-apply plugin: 'com.github.dcendents.android-maven'
-apply plugin: "com.jfrog.bintray"
```
This plugin can go

vbuberen replied:
What about the plugin above called `android-maven` ?

cortinico commented on build.gradle:
```diff
@@ -29,8 +29,6 @@ buildscript {
     dependencies {
         classpath "com.android.tools.build:gradle:$androidGradleVersion"
-        classpath "com.github.dcendents:android-maven-gradle-plugin:$androidMavenGradleVersion"
-        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:$gradleBintrayVersion"
```
This plugin can go

vbuberen commented:
Addressed your comments.

Note: we might need to consider switching to another plugin.
According to https://github.com/dcendents/android-maven-gradle-plugin new supported plugin is https://github.com/sky-uk/gradle-maven-plugin

Not an urgent change, but let's add it to roadmap.

vbuberen commented:
✔️
